src/main/io/gps.c

```diff
@@ -417,9 +419,40 @@ void gpsInitNmea(void)
                if (!atgmRestartDone) {
                    atgmRestartDone = true;
                    serialPrint(gpsPort, "$PCAS02,100*1E\r\n");  // 10Hz refresh rate
-                   serialPrint(gpsPort, "$PCAS10,0*1C\r\n");    // hot restart 
+                   serialPrint(gpsPort, "$PCAS10,0*1C\r\n");    // hot restart
                }
-               gpsData.state_position++;
+
```

Comment:
(the indentation is one off)

Reply:
It's wrong everywhere but I think a different PR would be better to fix these.

Reply:
IMO code below is indented 5 spaces, it should be in same column as `}` above (4 spaces)

Reply:
I fixed, but it's starting earlier in the code.
![image](https://user-images.githubusercontent.com/62965528/230334553-68b21a03-fc8f-4bc0-9e29-2a2f266bf9fa.png)